pages/home.html

```diff
@@ -235,7 +235,7 @@ <h2 class="text-center outer-heading"><i class="fa fa-users"></i> Join the Commu
   </div>
   <div class="white-box more-bottom">
     <div class="row">
-      <p>Have questions or need a hand? As a free and open-source project, our valued community of users and contributors from around the world are in the best position to help. Before diving in, we encourage you to read about <a href="/support/#staying-safe">staying safe</a>, our <a href="/support/#discussion-guidelines">discussion guidelines</a>, and our <a href="/code-of-conduct/">code of conduct</a> to help keep things positive and on-track. We welcome newcomers and returning users wanting to discuss Qubes and <a href="/doc/contributing/">seeking to contribute</a>.
+      <p>Have questions or need a hand? As a free and open-source project, our valued community of users and contributors from around the world are in the best position to help. Before diving in, we encourage you to read about <a href="https://doc.qubes-os.org/en/latest/introduction/support.html#staying-safe">staying safe</a>, our <a href="https://qubes-doc-rst.readthedocs.io/en/latest/introduction/support.html#discussion-guidelines">discussion guidelines</a>, and our <a href="/code-of-conduct/">code of conduct</a> to help keep things positive and on-track. We welcome newcomers and returning users wanting to discuss Qubes and <a href="/doc/contributing/">seeking to contribute</a>.
```

Comment:
href here should be https://doc.qubes-os.org/en/latest/introduction/support.html#discussion-guidelines and not https://qubes-doc-rst.readthedocs.io/en/latest/introduction/support.html#discussion-guidelines